.github/workflows/validate_linux_rt.yml

```diff
@@ -0,0 +1,34 @@
+name: Validate Linux RT Feed
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  validate_linux_rt:
+    name: Validate Linux RT Feed
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Repo
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Setup python3
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: Install build dependencies
+      run: python -m pip install -r python_build_requirements.txt
+
+    - name: Get previous tag
+      id: previoustag
+      run: >-
+        echo "::set-output name=tag::$(
+          git describe --abbrev=0 --tags ${{github.ref_name}}^
```

Comment:
The last tagged commit would not necessarily be an ancestor of the current commit. I.e., there could be cherry-picks into the previous release branch. Would this catch that? I think that's relatively common.

I expect it's possible to get all `v` tags in the repo and sort them by semver. Not sure if that's the best way to do it. Sorting by date would work in most cases, but potentially not patch releases.

Reply:
Yeah, I updated these to sort by semver like the step I have in the create_release action.